Dockerfile

```diff
@@ -1,6 +1,5 @@
 ARG node_version
-FROM node:$node_version
-
+FROM node:10
```

Comment:
Not working with node 8 but with 10 works fine.

Reply:
Is there a ticket to have versions other than 8 fixed up? If so, please link it here. The lack of 8 support isn't terrible, but failing on 12 is surprising.

Reply:
I found this old issue related and with similar error: https://github.com/medic/medic-couch2pg/issues/42
Maybe because of the dependencies in this project. 
I think @njogz might have fixed it for Node 8 (?) by setting a fix version number for 
`"pouchdb-core": "7.0.0",
    "pouchdb-adapter-http": "7.0.0"`

Reply:
The issue still happening, currently researching...

Reply:
I've fixed the build in node versions in this [PR](https://github.com/medic/medic-couch2pg/pull/73), code adapted to test in version 8, 10 and 12.